### ISSUE TITLE
clerk/md has double-nesting?

### ISSUE THREAD

**Issue by sritchie** [nextjournal/clerk#423]
I am using clerk/md to dynamically generate a shell command:

<img width="967" alt="image" src="https://user-images.githubusercontent.com/69635/219491040-26d5ded4-6b51-4d5d-ae58-13c1d9c9f868.png">

```clj
;; To create a new Clerk project based on
;; [`mafs/clerk`](https://github.com/mentat-collective/clerk-utils/tree/main/resources/clerk_utils/custom)
;; in a folder called `my-notebook-project`, run the following command:

^{::clerk/visibility {:code :hide}}
(clerk/md
(format "
```sh
clojure -Sdeps '{:deps {io.github.mentat-collective/mafs.cljs {:git/sha \"%s\"}}}' \\
-Tnew create \\
:template mafs/clerk \\
:name myusername/my-notebook-project
```" (docs/git-sha)))
```

and the rendered block has two scroll bars, and a narrower width than other code:

<img width="956" alt="image" src="https://user-images.githubusercontent.com/69635/219490958-3ce58986-d5bf-4116-9a61-0c4ebc5daa3e.png">

I still get a block with a single scroll bar if my generated markdown via `clerk/md` is narrower:

<img width="950" alt="image" src="https://user-images.githubusercontent.com/69635/219491221-f767f624-616b-4c8c-8bf1-b3e89ec09d79.png">

**Comment by zampino** [comment]
I can't reproduce the double nesting on main (at fad499407d979916d21b33cc7e46e73f7a485e37) but something is wrong with markdown code block margins (maybe after 94dfb96138cfc549c3e4c0948f6ebf71380a4676 ?). Trying with 

    ;; ```py
    ;; code
    ;; ```
    ;; text
    
    (clerk/md "```py
    code
    ```")
    
    (clerk/md
     (format "
    ```sh
    clojure -Sdeps '{:deps {io.github.mentat-collective/mafs.cljs {:git/sha "%s"}}}' \
    -Tnew create \
    :template mafs/clerk \
    :name myusername/my-notebook-project
    ```" "fad499407d979916d21b33cc7e46e73f7a485e37"))

gives

![CleanShot 2023-02-17 at 11 11 39](https://user-images.githubusercontent.com/1078464/219616590-107e3170-e8fe-4228-955c-1e7e31201379.gif)

**Comment by mk** [comment]
This has been fixed with d4a43dd.
